frontend/plugins/accounting_ui/src/modules/transactions/transaction-form/components/common/Summary.tsx

```
@@ -1,27 +1,31 @@
 import { Button } from 'erxes-ui';
+import { useAtom } from 'jotai';
 import { useWatch } from 'react-hook-form';
-import { TR_SIDES } from '../../types/constants';
-import { ITransactionGroupForm } from '../types/AddTransaction';
+import { TR_SIDES } from '../../../types/constants';
+import { followTrDocsState } from '../../states/trStates';
+import { ITransactionGroupForm } from '../../types/AddTransaction';
+
+const getSum = (trDocs: any[], sumDebit: number, sumCredit: number) => {
+  trDocs?.forEach((tr) => {
+    if (!(tr?.details && tr?.details[0])) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)